docs/UserGuide.md

```diff
@@ -76,15 +76,15 @@ Format: `help`
 
 Adds a person to the address book.
 
-Format: `add n/NAME e/EMAIL a/ADDRESS [p/PHONE] [t/TAG]…​`
+Format: `add n/NAME e/EMAIL r/ROLE a/ADDRESS c/COURSE [p/PHONE] [t/TAG]…​`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 A person can have any number of tags (including 0)
 </div>
 
 Examples:
-* `add n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01 c/CS2101`
-* `add n/Betsy Crowe t/friend e/betsycrowe@example.com a/Newgate Prison c/ST2334 t/criminal`
+* `add n/John Doe p/98765432 e/johnd@example.com r/STUDENT a/John street, block 123, #01-01 c/CS2101`
+* `add n/Betsy Crowe t/friend e/betsycrowe@example.com r/TA a/Newgate Prison c/ST2334 t/criminal`
```

Comment:
Same as above, need to change the address to school context. Also, can change `Betsy Crowe`'s tag to something other than `criminal`